### DISCUSSION TITLE
Game controller not working in Heroic while steam is running

### DISCUSSION THREAD

**Discussion by jeisom** [Heroic-Games-Launcher/HeroicGamesLauncher#2450] [Q&A]
Is this a problem others have run into?  If I have steam open, when playing the witcher 3 it doesn't/can't use the game controller. If I close steam, the controller works fine. I am on arch linux and I am using a Stratus Duo.

**Comment by chronohart** [comment]
Same thing is happening to me. I have Heroic Games Launcher installed by flatpak and Steam installed by deb. Running on Pop_OS. When Steam is open, my controller isn't recognized in HGL itself or in games launched from it. As soon as I close Steam, everything works again.

**Comment by Renari** [comment]
Steam implements it's own controller layer you can check what it is doing via launching big picture mode and going to your controller settings. I have suspicion that this is your issue and it would work if you launched heroic or the heroic installed game through steam as a non-steam application.

In big picture mode you can also disable all the controller support entirely which would make steam not take control of your pad when you plug it in, if you're using something other than an xinput controller the steam controller layer is actually really useful at translating your inputs to what games expect them to be.

**Comment by arielj** [comment]
Sounds like the problem here is steam and not heroic, if Steam hijacks the controller in some way and prevents us from detecting it there's nothing we can do.
